evap/evaluation/models.py

```diff
@@ -999,14 +988,6 @@ def unlogged_fields(self):
             "_participant_count",
         ]
 
-    @classmethod
-    def transform_log_action(cls, field_action):
```

Comment:
Nice, we can get rid of `transform_log_action` everywhere then! If you want, you can try it here, otherwise you or someone else can also do it in another PR later

Reply:
we should probably move this into a different PR. When I quickly tested it just now, it threw multiple issues.